state_management/csharp/sdk/order-processor/Program.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Dapr.Client;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading;
+using System.Text.Json;
+using System.Text;
+using System.Text.Json.Serialization;
+
+string DAPR_STORE_NAME = "statestore";
+var client = new DaprClientBuilder().Build();
+while(true) {
+    Random random = new Random();
+    var orderId = random.Next(1,1000).ToString();
+    var order = new Order(orderId);
+
+    // Save state into the state store
+    await client.SaveStateAsync(DAPR_STORE_NAME, orderId, order.ToString());
+
+    // Get state from the state store
+    var result = await client.GetStateAsync<string>(DAPR_STORE_NAME, orderId);
+    Console.WriteLine("Result after get: " + result);
+    
+    // Delete state from the state store
+    await client.DeleteStateAsync(DAPR_STORE_NAME, orderId);
+    Console.WriteLine("Order requested: " + order);
+    Console.WriteLine("Result: " + result);

```

Comment:
Still two lines.  should be one.

Also "Deleted order.".  This does not return a result.